build-test.sh

```diff
@@ -3,30 +3,14 @@
 initHostDistroRid()
 {
     __HostDistroRid=""
-    if [ "$__HostOS" == "Linux" ]; then
```

Comment:
@jashook I am sorry I've missed this PR, but this doesn 't look right. We cannot use portable build everywhere. Alpine Linux and CentOS / RHEL 6 need to use nono-portable builds specific for those.

Reply:
Do we publish non-portable alpine packages?

Reply:
I made the change as well for the CentOS scenario, our centOS packages are extremely out of date and restoring them will cause test failures.

Reply:
@jashook we do publish non-portable Alpine packages. As for CentOS 6 (please note that it is only for the version 6, newer versions use portable build), the RID is in fact rhel.6. See the code you've deleted below, you'll see that. And we publish rhel.6 packages that are up to date.